test/helm-test/main.go

```diff
@@ -252,7 +252,11 @@ func doMain() int {
 		xmlWrap(fmt.Sprintf("Helm Test %s", path.Base(chartPath)), func() error {
 			o, execErr := output(exec.Command(helmPath, "test", rel))
 			if execErr != nil {
-				return fmt.Errorf("%s Command output: %s", execErr, string(o[:]))
+				l, logsErr := output(exec.Command(kubectlPath, "logs", rel, "--namespace", ns))
```

Comment:
This does `kubectl logs $RELEASE` but that wont necessarily map to any pods afaik.

Reply:
@mattfarina did a similar thing for the pre-submit tests here:
https://github.com/kubernetes/charts/blob/master/test/changed.sh#L48

Reply:
This isn't going to work if a pod has more than one container. logs needs to be fetched for each container in a pod.